app/assets/javascripts/app/service-provider-form.js

```diff
@@ -19,13 +21,44 @@ $(function(){
     }
   }
 
+  var issuer_protocol_name = function(id_protocol){
+    switch(id_protocol){
+      case 'openid_connect':
+        return 'openidconnect';
+        break;
```

Comment:
Looks like code climate is mad at the `break` after a `return` because it's unreachable (since the `return` leaves the function before the `break`, can we remove these?